app/src/main/java/fr/free/nrw/commons/EventLog.java

```diff
@@ -109,7 +109,9 @@ public void log(boolean force) {
                 return; // User has disabled tracking
             }
             LogTask logTask = new LogTask();
-            Utils.executeAsyncTask(logTask, this);
+            logTask.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR, this);
```

Comment:
Oops sorry, I have no idea why I replicated it three times. The rebase reflects these changes.